frontend/public/components/factory/table.tsx

```diff
@@ -580,36 +582,38 @@ export const Table: React.FC<TableProps> = ({
   const children = mock ? (
     <EmptyBox label={label} />
   ) : (
-    <TableWrapper virtualize={virtualize} ariaLabel={ariaLabel} ariaRowCount={ariaRowCount}>
-      <PfTable
-        cells={columns}
-        rows={
-          virtualize
-            ? []
-            : Rows({
-                componentProps,
-                selectedResourcesForKind,
-                customData,
-              })
-        }
-        gridBreakPoint={gridBreakPoint}
-        onSort={onSort}
-        onSelect={onSelect}
-        sortBy={sortBy}
-        className="pf-m-compact pf-m-border-rows"
-        role={virtualize ? 'presentation' : 'grid'}
-        aria-label={virtualize ? null : ariaLabel}
-      >
-        <TableHeader role="rowgroup" />
-        {!virtualize && <TableBody />}
-      </PfTable>
-      {virtualize &&
-        (scrollNode ? (
-          renderVirtualizedTable(scrollNode)
-        ) : (
-          <WithScrollContainer>{renderVirtualizedTable}</WithScrollContainer>
-        ))}
-    </TableWrapper>
+    <div className={virtualize ? 'pf-c-scrollablegrid' : ''}>
```

Comment:
Recommend using https://github.com/JedWatson/classnames instead of a ternary to avoid passing an empty string.